code/game/machinery/vending_types.dm

```diff
@@ -316,22 +316,22 @@
 	icon_state = "cigs"
 	vend_id = "smokes"
 	products = list(
-		/obj/item/storage/fancy/cigarettes/rugged = 2,
-		/obj/item/storage/fancy/cigarettes = 3,
-		/obj/item/storage/fancy/cigarettes/dromedaryco = 2,
-		/obj/item/storage/fancy/cigarettes/nicotine = 2,
-		/obj/item/storage/fancy/cigarettes/pra = 2,
-		/obj/item/storage/chewables/rollable/bad = 2,
-		/obj/item/storage/chewables/rollable = 2,
-		/obj/item/storage/chewables/rollable/fine = 2,
-		/obj/item/storage/chewables/rollable/nico = 2,
-		/obj/item/storage/chewables/tobacco/bad = 2,
-		/obj/item/storage/chewables/tobacco = 2,
-		/obj/item/storage/chewables/tobacco/fine = 2,
-		/obj/item/storage/fancy/chewables/tobacco/nico = 2,
-		/obj/item/storage/cigfilters = 5,
-		/obj/item/storage/fancy/cigpaper = 3,
-		/obj/item/storage/fancy/cigpaper/fine = 2,
+		/obj/item/storage/fancy/cigarettes/rugged = 8,
+		/obj/item/storage/fancy/cigarettes = 8,
+		/obj/item/storage/fancy/cigarettes/dromedaryco = 6,
+		/obj/item/storage/fancy/cigarettes/nicotine = 4,
+		/obj/item/storage/fancy/cigarettes/pra = 6,
+		/obj/item/storage/chewables/rollable/bad = 8,
+		/obj/item/storage/chewables/rollable = 6,
+		/obj/item/storage/chewables/rollable/fine = 6,
+		/obj/item/storage/chewables/rollable/nico = 4,
+		/obj/item/storage/chewables/tobacco/bad = 8,
+		/obj/item/storage/chewables/tobacco = 6,
+		/obj/item/storage/chewables/tobacco/fine = 6,
+		/obj/item/storage/fancy/chewables/tobacco/nico = 4,
```

Comment:
Oh yeah, _you still haven't explained_ **why** you've increased the stock tenfold. **For everything.**

Hell, even the med-vendor right below this machine's biggest stock is 11, and those are for tiny, disposable syringes.

Reply:
There is only two public vendors. The shit cigs are cheap and common, as how it works in real life.

Reply:
...then add more vendors?

Reply:
No thanks.